modules/programs/feh.nix

```diff
@@ -0,0 +1,41 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.feh;
+
+  disableBinding = func: key: func;
+  enableBinding = func: key: "${func} ${key}";
+
+in
+
+{
+  options.programs.feh = {
+    enable = mkEnableOption "feh - a fast and light image viewer";
+
+    keybindings = mkOption {
+      default = {};
+      type = types.attrs;
+      example = { zoom_in = "plus"; zoom_out = "minus"; };
+      description = ''
+        Set keybindings.
+        See https://man.finalrewind.org/1/feh/#x4b455953 for
```

Comment:
For some docbook magic you can use `<link xlink:href="https://man.finalrewind.org/1/feh/#x4b455953" />` here. Otherwise looking fine 👍